webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9378,6 +9378,11 @@ var userOverride = {
                 var attachmentBox = 'n-vis';
                 var kmAttchMsg = '';
                 let isUserMsg = true;
+
+                if (msg?.message) {
+                    msg.message = window.DOMPurify.sanitize(msg.message);
```

Comment:
what is the purpose of appending window. here
is it because of iFrame thing?

Reply:
window is referring to the iframe context here. 
This is a similar `window.Dom || Dom` but following the widget code convention